napari/_vispy/overlays/_interaction_box_constants.py

```diff
@@ -0,0 +1,15 @@
+class Box:
```

Comment:
This looks duplicated below do we need another copy?

Reply:
Yes, that's duplicated. It is not really needed for the model, so I removed it from there.